modules/navier_stokes/include/fvkernels/PINSFVEnergyTimeDerivative.h

```diff
@@ -0,0 +1,31 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "FVTimeKernel.h"
+
+class PINSFVEnergyTimeDerivative : public FVTimeKernel
```

Comment:
doxygen please

Reply:
ok adding them to all fvbcs

Reply:
I think this might be a hint that I did not properly doxygen all the INSFV classes I added. If so then touche :smile: